pom.xml

```diff
@@ -130,5 +129,17 @@
             <version>1.0.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins</groupId>
+            <artifactId>configuration-as-code</artifactId>
+            <version>1.19</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.jenkins.configuration-as-code</groupId>
+            <artifactId>configuration-as-code-support</artifactId>
```

Comment:
`configuration-as-code-support` is TRASHED, please remove it.

Reply:
Addressed here: https://github.com/imaffe/external-workspace-manager-plugin/pull/4